ui/views.py

```diff
@@ -266,7 +265,10 @@ def serve_static_assets(request, path):
     # first check if the user has access to the file
     media_path = os.path.join(STATIC_ASSET_PREFIX, path)
     file_path = os.path.join(settings.MEDIA_ROOT, media_path)
-    static_asset = get_object_or_404(StaticAsset, asset=media_path)
+    static_assets = StaticAsset.objects.filter(asset=media_path)
```

Comment:
now yes: there is not check during the import that the path should be unique for a course id